src/main/java/org/opentripplanner/model/TransferPriority.java

```diff
@@ -0,0 +1,27 @@
+package org.opentripplanner.model;
+
+/**
+ * This is equivalent to NeTEx InterchangeWeightingEnumeration. It can specify the priority
+ * of transfers both at a Stop level or Interchange/Transfer level. OTP currently only supports
+ * TransferPriority at the Stop level.
+ *
+ * GTFS currently does not support transfer priority.
+ */
+public enum TransferPriority {
```

Comment:
We discussed renaming this enum and its values to reflect the fact that we are actually prioritizing stops, not only transfers. So the priorities would apply even on access/egress, not only at transfers. Ideally we'll have an enum that can be reused in the future for prioritizing transfers themselves (as well as stops).

Reply:
Based on the discussion we had I rename the enum and its values.